sklearn_bench/nusvc.py

```diff
@@ -43,33 +43,44 @@ def main():
 
     if params.probability:
         state_predict = 'predict_proba'
-        accuracy_type = 'log_loss'
         clf_predict = clf.predict_proba
-
-        def metric_call(x, y):
-            return bench.log_loss(x, y)
+        y_proba_train = clf_predict(X_train)
```

Comment:
I think it makes no sense for svm's to display metrics log_loss, roc_auc, because by default he don't return proba

Reply:
If user want to turn on probability then he will see these metrics